### PR TITLE
Added Library Updates descriptions

### PR DESCRIPTION
**Pull request by loocool2**

### CODE REVIEW

File: src/help/guides/library-updates.md [hunk 1]
```diff
@@ -4,4 +4,11 @@ lang: en-US
 ---
 
 # Library updates
```
Comment (Soitora) [inline src/help/guides/library-updates.md]: This page is now purely called "Updates" (which outdates a whole lot of content).

File: src/help/guides/library-updates.md [hunk 2]
```diff
@@ -4,4 +4,11 @@ lang: en-US
...
-:construction: :construction_worker_man: W.I.P.
+
+This page will display all chapters recently updated, sorted by the time they were found and added. Chapters will be seperated by the day they were found, in ascending order. It is possible to scroll as far back as desired
+
+## 3 dot menu
```
Comment (Soitora) [inline src/help/guides/library-updates.md]: This is also called "Overflow menu"
I've referred to it with icon before as `**<MaterialIcon icon-name="more_vert"/> Overflow menu**`

File: src/help/guides/library-updates.md (hunk 1 above)
Comment (arkon) [reply src/help/guides/library-updates.md]: We can update it once stable is updated.

File: src/help/guides/library-updates.md [hunk 3]
```diff
@@ -4,4 +4,11 @@ lang: en-US
 ---
 
 # Library updates
-:construction: :construction_worker_man: W.I.P.
+
+This page will display all chapters recently updated, sorted by the time they were found and added. Chapters will be seperated by the day they were found, in ascending order. It is possible to scroll as far back as desired
```
Comment (arkon) [inline src/help/guides/library-updates.md]:
```diff
- This page will display all chapters recently updated, sorted by the time they were found and added.
+ This page displays all new chapters sorted by when they were added.
```

I feel like the rest of it isn't really necessary.

File: src/help/guides/library-updates.md (hunk 2 above)
Comment (loocool2) [reply src/help/guides/library-updates.md]: I guess you can replace it with overflow menu with the icon next to it

**Pull request merged**